### PR TITLE
Page header trilogy

### PR DESCRIPTION
**Pull request by jamesgorrie**
3rd in the trilogy of four, the next will be to deprecate everything, but I will do that as a last step once these are merged in.

### CODE REVIEW

File: common/views/components/BasePage/BookPage.js [hunk 1]
```diff
@@ -45,24 +43,25 @@ const BookPage = ({ book }: Props) => {
...
+  const breadcrumbs = {
+    items: [{
+      text: 'Books',
       url: '/books'
-    }]} />}
-    DateInfo={null}
-    Description={
+    }]
+  };
+  const NewHeader = <PageHeader
```
Comment (gestchild) [inline common/views/components/BasePage/BookPage.js]: Why is this NewHeader and not just Header?

Comment (jamesgorrie) [reply common/views/components/BasePage/BookPage.js]: There's something already in Header, which seems to be `SiteHeader`

Comment (gestchild) [reply common/views/components/BasePage/BookPage.js]: Ignore me, you changed it in the next commit :-)

**Comment by jamesgorrie** [comment]
Thanks @gestchild - sorry for the 🐘 PR.

**Pull request merged**